extensions/positron-connections/src/test/connection.test.ts

```diff
@@ -17,6 +17,14 @@ suite('Connections pane works for R', () => {
 
 	test('Can list tables and fields from R connections', async () => {
```

Comment:
Do the tests need to be updated to use the new pane?

Reply:
I have opened https://github.com/posit-dev/positron/issues/6752 to track porting these tests. However since the core service now lives in Positron Core we need a different strategy as they can't be simple extension tests. We already have e2e tests that run for the new connections pane though.